### PR TITLE
Posts table

### PR DESCRIPTION
**Pull request by dcwds**
The following changes have been made according to the AC:
- posts table is shown when cell in heatmap is clicked and not shown otherwise
- posts table contains post title, time, score, comment count, and author
- post title and author link to their respective pages in a new tab
- author is not linked if the account is deleted
- posts are sorted by the time-of-day they were created, from earliest to latest

### CODE REVIEW

**Comment by mentor-tara** [comment]
Great job :rocket: I'm a bit busy at the moment but I'll leave a review soon.

In the meantime, consider starting with the next task already to practice the professional Git workflow. [If you need a refresher, feel free to follow this section in the course again.](https://profy.dev/project/reddit-timer/while-waiting)[](ckmadkmzs3nk50b35tdpzy2sb)

**Review by mentor-tara — CHANGES_REQUESTED** [review]
Before I approve this PR I'd like to share a few tips with you.

**Please go through the list below and review your own code first.** Imagine you're the reviewer or a person who didn't see the code before. Ask yourself if everything is clearly written and understandable. Are components and variables properly named?

Reviewing your own code with this mindset is an important practice that will help you learn a lot and write high-quality code.

To make it a bit easier for you here are the tips I promised. Some have a link :link: attached where you can find more information. I'd recommend reading the **bold tips** in any case. Feel free to check the boxes when you think your code fits.

**General:**
- [ ] **Don't duplicate data** [:link:](https://profy.dev/project/reddit-timer/review-tips/dont-duplicate-data)
- [ ] Throw away data that is not used [:link:](https://profy.dev/project/reddit-timer/review-tips/throw-away-unused-data)
- [ ] Rename data fields [:link:](https://profy.dev/project/reddit-timer/review-tips/rename-data-fields)
- [ ] **Make sure variable names are descriptive** [:link:](https://profy.dev/project/reddit-timer/review-tips/variable-naming)

**Styles:**
- [ ] Fixed width is only necessary for title and author [:link:](https://profy.dev/project/reddit-timer/review-tips/fixed-width-for-table-columns)
- [ ] **Refrain from styling children with CSS selectors** [:link:](https://profy.dev/project/reddit-timer/review-tips/css-selectors)

Once you're done with reviewing your own code feel free to check the [example implementation](https://profy.dev/project/reddit-timer/code-examples/posts-table-f1f46148-0b85-4885-9b8e-998bc03ff66d). **But remember: It's an important exercise to review your code first on your own.**

When you think this PR is ready to be merged just let me know by requesting another review. I'll approve right away.[](ckmg033s80apg0c82119d246z)

**Review by mentor-tara — APPROVED** [review]
Great work! :tada::tada::tada: You can merge the PR now.[](ckmadozmo3npp0b353eayp9qn)

File: src/components/page-search/page-search.test.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ describe('heatmap', () => {
   it('loads posts into heatmap via subreddit in URL', async () => {
     setup('/search/reactjs');
 
-    await waitForElementToBeRemoved(() => screen.getByLabelText(/loading/i));
+    await waitForElementToBeRemoved(() => screen.getByLabelText(/loading/i), { timeout: 10000 });
```
Comment (jkettmann) [inline src/components/page-search/page-search.test.js:68]: It is interesting that this takes so long. Are you sure that the mock handlers are called? This looks suspiciously like it's sending real requests

**Pull request merged**